Basic-Car-Maintenance/Shared/Models/Vehicle.swift

```diff
@@ -18,17 +22,23 @@ struct Vehicle: Codable, Identifiable, Hashable {
     let color: String?
     let vin: String?
     let licensePlateNumber: String?
+    var displayRepresentation: DisplayRepresentation { DisplayRepresentation(title: "\(name)") }
+    
+    static var defaultQuery = VehicleQuery()
+    static var typeDisplayRepresentation = TypeDisplayRepresentation(name: "Vehicle")
     
-    init(id: String? = nil,
-         userID: String? = nil,
-         name: String,
-         make: String,
-         model: String,
-         year: String? = nil,
-         color: String? = nil,
-         vin: String? = nil,
-         licensePlateNumber: String? = nil) {
-        self.id = id
+    init(
+        id: String = "",
+        userID: String? = nil,
+        name: String,
+        make: String,
+        model: String,
+        year: String? = nil,
+        color: String? = nil,
+        vin: String? = nil,
+        licensePlateNumber: String? = nil
+    ) {
+        self.documentID = id
```

Comment:
The documentID shouldn't be set from the id, because it would always be empty

this gets set properly when fetching from Firebase